### ISSUE TITLE
Can't download youtube shorts video

### ISSUE THREAD

**Issue by inmessi** [ytdl-org/youtube-dl#31080]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
PASTE VERBOSE LOG HERE
```

## Description

WRITE DESCRIPTION HERE

**Comment by inmessi** [comment]
I tried to download youtube shorts video but it showed:
```
ERROR: Unable to recognize tab page!
```

**Comment by dirkf** [comment]
> I've searched the bugtracker for similar bug reports including closed ones

Obvs not.

Duplicate of #30269, fixed in git master by PR #30577.
